types/tektelic/edoctorV1/lifecycle.schema.json

```
@@ -0,0 +1,21 @@
+{
+  "$id": "https://akenza.io/tektelic/edoctorV1/lifecycle.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Lifecycle",
+  "type": "object",
+  "topic": "lifecycle",
+  "processingType": "uplink_decoder",
+  "properties": {
+    "batteryLevel": {
+      "title": "Battery charge",
+      "type": "number",
+      "minimum": 0,
```

As this is a number, we could use `0.0` to highlight the possibility to support floating-point numbers. But really a nitpick!

```suggestion
      "minimum": 0.0,
```